### PR TITLE
Add conversions of code/coding etc to System types

### PR DESCRIPTION
**Pull request by ewoutkramer**
Added functions for converting the POCO bindeable types to the new System bindeable types:

* Code => System.Code
* CodeOfT => System.Code
* Coding => System.Code
* CodeableConcept => System.Concept

### CODE REVIEW

File: src/Hl7.Fhir.Support.Tests/Model/ConceptTests.cs [hunk 1]
```diff
@@ -6,7 +6,10 @@
  * available at https://raw.githubusercontent.com/FirelyTeam/firely-net-sdk/master/LICENSE
  */
 
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Support.Tests.Utils;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Linq;
 using P = Hl7.Fhir.ElementModel.Types;
 
 namespace Hl7.FhirPath.Tests
```
Comment (marcovisserFurore) [inline src/Hl7.Fhir.Support.Tests/Model/ConceptTests.cs:15]: You did not change the namespace, but I am very surprised that the namespace is here `Hl7.FhirPath.Tests` . Should we not change that?

**Review by marcovisserFurore — COMMENTED** [review]
See my comment

File: src/Hl7.Fhir.Support.Tests/Model/ConceptTests.cs (hunk 1 above)
Comment (ewoutkramer) [reply src/Hl7.Fhir.Support.Tests/Model/ConceptTests.cs:15]: Huh? Yeah, why not, it's just a test, we can change it!

File: src/Hl7.Fhir.Support.Tests/ElementModel/AnyTests.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@
 using System.Collections.Generic;
 using P = Hl7.Fhir.ElementModel.Types;
 
-namespace Hl7.FhirPath.Tests
+namespace Hl7.Fhir.Model.Tests
```
Comment (marcovisserFurore) [inline src/Hl7.Fhir.Support.Tests/ElementModel/AnyTests.cs:13]: The rest is named Hl7.Fhir.ElementModel.Tests

**Pull request merged**